src/base/bittorrent/torrentinfo.cpp

```diff
@@ -61,7 +61,7 @@ namespace
         {
             if (QDir::isAbsolutePath(filePath)) continue;
 
-            const auto filePathElements = filePath.splitRef('/');
+            const auto filePathElements = QStringView(filePath).split(u'/');
```

Comment:
>why not use `QLatin1Char {'/'}`?

Because `u'/'` is shorter (much). I like it more since I found this way. And I can't see any disadvantage of using it.

Reply:
I think there is a (negligible) drawback: the `u` prefixed strings would be stored in 16-bit as opposed in usual 8-bit which makes the binary slightly larger.

Reply:
I don't think it will increase much, so we can really neglect this disadvantage. Otherwise, we should require the use of QLatin1Char.